File: RSPAClient/Atomic/app/Entities/MT_POU_CASE_TRACK_HISTORY.ts
export class MT_POU_CASE_TRACK_HISTORY {
    public ID: number;
    public CASE_ID: string;
    public PREF_LIST_ID: string;
    public PROCEDURE_CODE: string;
    public PATIENT_ID: string;
    public DEPARTMENT_ID: string;
    public CASE_STATUS?: number;
    public UPDATE_USER_ID: string;
    public WORKSTATION_MAC_ADDRESS: string;
    public UPDATE_DT_TIME:  Date;
    public CASE_PICK_STATUS?: number;
    public TRANSACTION_ID?: number;
    public CHARGE_CAPTURE_TRANS_ID?: number;
    public PERFORM_DATE?: Date;
    public COMMENTS: string;
}

File: RSPAClient/Core/AtPar.POCOEntities/MT_POU_CASE_TRACK_HISTORY.cs
using System;
using System.Collections.Generic;

namespace AtPar.POCOEntities
{
    public partial class MT_POU_CASE_TRACK_HISTORY
    {
        public long ID { get; set; }
        public string CASE_ID { get; set; }
        public string PREF_LIST_ID { get; set; }
        public string PROCEDURE_CODE { get; set; }
        public string PATIENT_ID { get; set; }
        public string DEPARTMENT_ID { get; set; }
        public Nullable<short> CASE_STATUS { get; set; }
        public string UPDATE_USER_ID { get; set; }
        public string WORKSTATION_MAC_ADDRESS { get; set; }
        public System.DateTime UPDATE_DT_TIME { get; set; }
        public Nullable<short> CASE_PICK_STATUS { get; set; }
        public Nullable<int> TRANSACTION_ID { get; set; }
        public Nullable<int> CHARGE_CAPTURE_TRANS_ID { get; set; }
        public Nullable<System.DateTime> PERFORM_DATE { get; set; }
        public string COMMENTS { get; set; }
    }
}
